src/web/components/SlotBodyEnd.tsx

```diff
@@ -168,96 +163,60 @@ const MemoisedInner = ({
             contributionsServiceUrl,
             isSensitive,
         });
-        getBodyEnd(contributionsPayload, `${contributionsServiceUrl}/epic`)
+
+        window.automat = {
```

Comment:
Is there no way to pass this directly to the component? If not maybe worth having this as `window.guardian.automat`? Ideally having this defined in `DCRBrowserDocumentData` type object in `index.d.ts` would be great, but I am not sure how best to achieve this is

Reply:
Initially (for the simple prototype), it worked as you suggest - passing React as an explicit dependency. But It gets tricky when you have nested components as you want to develop them in the usual way (import react, etc.) so that it's easy to use them with other tooling, such as storybook and for familiarity - basically to keep them as regular React functions. If you force react/emotion etc as direct arguments, you have to depart from this.

I agree exporting on the window feels sad. Nesting under `window.guardian`makes sense though.